pytorch_blade/tests/torchscript/shape_analysis.graph

```diff
@@ -0,0 +1,48 @@
+// RUN: shape_analysis_tool <%s | FileCheck %s
+
+// CHECK-LABEL: graph
+graph(%p1 : Float(48, 128, 768, requires_grad=0),
+      %p2 : Float(1, 128, 768, requires_grad=0),
+      %p3 : int):
+// CHECK: Float(*, *, *) = aten::add(%p1, %p2, %p3)
+  %1 : Tensor = aten::add(%p1, %p2, %p3)
+  return (%1)
+
+// CHECK-LABEL: graph
+graph(%p1 : Float(1, 512, requires_grad=0, device=cpu)):
+  %1 : int = prim::Constant[value=0]()
+  %2 : int = prim::Constant[value=1]()
+// CHECK: Float(1, 512, requires_grad=0, device=cpu) = aten::slice(%p1, %1, %1, %2, %2)
+  %3 : Tensor = aten::slice(%p1, %1, %1, %2, %2)
+  return (%3)
+
+// CHECK-LABEL: graph
+graph(%p1 : Float(*, *, *, requires_grad=0),
+      %p2 : Float(*, *, *, requires_grad=0),
+      %p3 : int):
+// CHECK: Float(*, *, *) = aten::add(%p1, %p2, %p3)
+  %1 : Tensor = aten::add(%p1, %p2, %p3)
+  return (%1)
+
+// CHECK-LABEL: graph
+graph(%p1 : Float(*, *, requires_grad=0, device=cpu)):
```

Comment:
Need to configure `stride` ? ShapeAnalysisPass uses [isComplete()](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/jit_type.h#L285) to check static shape or not.

Reply:
OK. I know why the concrete shapes can't be propagated now. IMO, our shape analysis pass will not depends on strides setting.

Reply:
And we should fix it in another PR.